1주차/[Lv.3]_이중우선순위큐/주다현.js

```diff
@@ -0,0 +1,19 @@
+function solution(operations) {
+  let answer = [],
+    Q = [];
```

Comment:
```suggestion
   const answer = [];
   const Q = [];
```
```javascript
const Q = []; // before
const queue = []; // after
```
`Q`보다는 `queue`라고 이름을 짓는 것은 어떨까요?? 🤔
조금 귀찮을 수 있지만 명확하게 하는 것이 나쁘지는 않다고 생각합니다! 😄